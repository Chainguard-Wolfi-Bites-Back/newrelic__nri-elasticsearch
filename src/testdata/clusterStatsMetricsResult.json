{
	"cluster_name": "es6-dev-cluster",
	"status": "yellow",
	"timed_out": false,
	"number_of_nodes": 6,
	"number_of_data_nodes": 3,
	"active_primary_shards": 10,
	"active_shards": 30,
	"relocating_shards": 0,
	"initializing_shards": 0,
	"unassigned_shards": 7,
	"delayed_unassigned_shards": 0,
	"number_of_pending_tasks": 0,
	"number_of_in_flight_fetch": 0,
	"task_max_waiting_in_queue_millis": 0,
	"active_shards_percent_as_number": 81.08108108108108
}