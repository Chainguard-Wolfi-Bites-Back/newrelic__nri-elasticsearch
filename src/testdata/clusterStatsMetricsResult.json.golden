{"cluster.dataNodes":3,"cluster.nodes":6,"cluster.status":"yellow","displayName":"es6-dev-cluster","entityName":"es-cluster:es6-dev-cluster","event_type":"ElasticsearchClusterSample","shards.active":30,"shards.initializing":0,"shards.primaryActive":10,"shards.relocating":0,"shards.unassigned":7}